src/main/java/frc/robot/subsystems/Scoring.java

```diff
@@ -94,24 +88,18 @@ public double getPosition() {
         return leftScoringElevatorEncoder.getPosition();
     }
 
-    public void runLoader(double speed) {
-        speed = MathUtil.clamp(speed, ScoringConstants.minSpeed, ScoringConstants.maxSpeed);
-        leftLoadingMotor.set(speed);
+    public void runRoller() {
+        rollerMotor.set(ScoringConstants.rollerSpeed);
     }
 
-    public void runRoller(double speed) {
-        speed = MathUtil.clamp(speed, ScoringConstants.minSpeed, ScoringConstants.maxSpeed);
-        rollerMotor.set(speed);
+    public void rollerShoot() {
+        rollerMotor.set(-ScoringConstants.rollerSpeed);
     }
 
     public void stopRoller() {
         rollerMotor.stopMotor();
     }
 
-    public void stopLoader() {
-        leftLoadingMotor.stopMotor();
-    }
-
     public void setP(double p) {
```

Comment:
Do we need to set I and D values for the PID?

Reply:
These functions are useful for PID tuning because they can be wired to dashboard controls to tun the values without re-compiling and deploying